sanskrit_parser/parser/sandhi.py

```diff
@@ -25,10 +25,10 @@
 
 .. code:: python
 
-    >>> from sanskrit_parser.lexical_analyzer.sandhi import Sandhi
+    >>> from sanskrit_parser.parser.sandhi import Sandhi
     >>> sandhi = Sandhi()
-    >>> word1 = SanskritObject('te')
-    >>> word2 = SanskritObject('eva')
+    >>> word1 = SanskritImmutableString('te')
```

Comment:
Why not use SanskritNormalizedString to allow "loose" input (i,e, not strict_io) ?

Reply:
Since that's a programmatic example, we already know what we want, so ImmutableString is probably best in this narrow case. I agree with your point elsewhere that this hierarchy is confusing.